### DISCUSSION TITLE
Tooltip for flux:row and flux:cell

### DISCUSSION THREAD

**Discussion by ultimateBusiness** [livewire/flux#305] [Feature Requests]
Please, add tooltip to flux:row and flux:cell.

**Comment by joshhanley** [comment]
@ultimateBusiness the problem is with semantic HTML, a table must be structured correctly, so we can't wrap a tooltip element around a cell or row. So if you need this, I would recommend just adding the tooltip inside a cell wrapping around an element inside the cell.
